src/lib/Zikula/Bundle/CoreInstallerBundle/composer.json

```diff
@@ -14,9 +14,17 @@
     "require": {
         "php": ">=7.2.0",
         "symfony/framework-bundle": "4.*",
+        "symfony/console": "^4.3",
+        "symfony/requirements-checker": "dev-patch-1 as 1.*",
         "matthiasnoback/symfony-console-form": "3.*",
         "zikula/wizard": "3.*"
     },
+    "repositories": [
+        {
+            "type": "git",
+            "url": "https://github.com/craigh/requirements-checker"
```

Comment:
What is the reason for using your fork? You did no changes there so far.

Reply:
Found it: https://github.com/craigh/requirements-checker/commit/7149dcfdac06eec8c6f7efc0b1e2ecb4e0d25261